tests/README.md

```diff
@@ -0,0 +1,74 @@
+# Test documentation
+
+## List of scripted integration tests
+
+1. Up
+    * file: up.bash
+    * purpose: test if the SonarQube server is UP
+1. Plugin check
+    * file: check_plugins.bash
+    * purpose: check that the plugins listed in the README are installed on the server with the right version
+1. Quality Gate check
+    * file: check_qg.bash
+    * purpose: check that the CNES Quality Gate is available on the server and is set as default
+1. Quality Profiles check
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `4`, was `1`](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6196008)